### DISCUSSION TITLE
Flow for mouse tracker ?

### DISCUSSION THREAD

**Discussion by cgidesign-de** [wojtryb/Shortcut-Composer#50] [Ideas]
Hi,

found your shortcut composer and like it a lot.
Thanks for making it !!

I already have one feature request:
Please add "flow" to the mouse tracker functionality.
I am not that often changing the opacity but instead the flow.

As I understand the mouse tracker, only opacity + size can be tracked.
An option to choose flow + size would be great.

Thanks and regards.

**Comment by wojtryb** [comment]
Hi, and thanks for using my plugin!

Shortcut Composer was written with configurability like this in mind. There is no GUI for that, but editing required is rather trivial.
Open `Settings > Manage Resources > Open Resource Folder > pykrita > shortcut_composer > actions.py` file in any text editor, and search (<kbd>ctrl</kbd>+<kbd>f</kbd>) for `Scroll brush size or opacity` action definition.

All you need to do to get what you ask for, is to replace the `OpacityController` with `FlowController`:
```diff
    templates.CursorTracker(
        name="Scroll brush size or opacity",
        horizontal_slider=Slider(
            controller=controllers.BrushSizeController(),
            values=[
                1, 2, 3, 4, 5, 6, 7, 8, 9,
                10, 12, 14, 16, 20, 25, 30, 35, 40, 50, 60, 70, 80,
                100, 120, 160, 200, 250, 300, 350, 400, 450,
                500, 600, 700, 800, 900, 1000
            ],
            sensitivity_scale=2,
        ),
        vertical_slider=Slider(
-           controller=controllers.OpacityController(),
+           controller=controllers.FlowController(),
            values=Range(10, 100),
            sensitivity_scale=10,
        ),
    ),
```
Let me know if you have any problems with that, or when you managed to do the trick :)

**Reply by cgidesign-de** [reply]
Thanks, that did the trick. 👍
